CHARTER.md

```diff
@@ -0,0 +1,104 @@
+
+While the Governing Board’s purpose is not to handle technical topics in general, the Board retains the authority over all decisions the MapLibre Projects make, and may overrule the Technical Steering Committee in case of a dispute.
+
+No more than one person on the Governing Board shall be employed or contracted by a single Third-party organization.
+
+The Board shall have at least the following roles:
+
+ * Treasurer: Takes care of financial transactions and accounting.
+ * Secretary: Writes and publishes meeting notes, manages Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
+
+Voting Members can nominate other Community members to receive Voting membership in acknowledgement of their contributions to the MapLibre Projects. The Voting Members then vote on granting the nominee a Voting membership. 
+
+Nominations for new Voting Members are open up to two weeks before the election of the Governing Board and the vote on granting Voting Membership to somebody takes place one week before the election of the Governing Board.
+
+### Governing Board Election
+Nomination process:
+
+ * A person can nominate themselves as a candidate for the Governing Board.
+ * Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
+
+ * Governing Board elections take place every year.
+ * Each Voting Member can vote for up to 5 nominees with ranked voting principle.
+ * The nominee with the largest number of votes gets appointed, the nominee with the second largest number of votes gets appointed, and so on.
+ * In case there are multiple nominees per Third-party organization, only the one with the largest number of votes gets appointed.
+ * In case of a tie where more nominees than remaining seats receive an equal number of votes, the vote shall be repeated for the remaining seats.
+
+### Terms of Governing Board Members
+Governing Board members will serve for a one-year term.
+
+### Governing Board Member Remuneration
+Governing Board members receive no remuneration by the MapLibre Organization for their services on the Board, other than reimbursement for the out-of-pocket expenses incurred when carrying out activities on behalf of the Governing Board. Such expenses must be pre-authorized by at least two other board members.
```

Comment:
I believe this is not clear enough -- this paragraph makes it ambiguous if someone can sit on the board - vote for fund distributions (i.e. grants), and later receive those grants, or even be contracted by the board to perform some task.   I think it creates a serious conflict of interest, as they would essentially vote to pay themselves.

That said, I am fully aware that if someone indirectly benefits financially from the MapLibre projects, they also cannot be fully impartial. Unfortunately, this includes pretty much everyone because all of us have some sort of interest in MapLibre's success one way or another.

Reply:
Thanks for sharing this concern @nyurik. I would say that activity on the Board cannot be paid, but work related to maintenance of MapLibre Projects by a Governing Board member can be paid. But then there is of course that conflict of interest as you have mentioned. I don't know how to solve this problem...

Reply:
Agreed that board membership is not remunerated (except _maybe_ in the edge case of some token gesture voted on by the voting members).

> work related to maintenance of MapLibre Projects by a Governing Board member can be paid.

I think the only way this would be acceptable is if the member receiving the funds did not participate in the vote to approve the funds, and even then it opens up the door for "you scratch my back, I'll scratch yours" arrangements which could reduce  the integrity of the board.

I'm inclined to disallow the governing board members from receiving funds, with the possible exception of something directly related to work that anyone could have done (e.g., bounties).

Reply:
Note that there is even worse cases -- something like "I am interested in the tile serving, and want to work on that, therefor I will vote to reduce the funding to Native because this way there will be more funding for me".  All this is hypothetical, but it may cause issues.

Reply:
A few important points were raised by @maxammann & @mojodna yesterday in the TSC meeting.

1) Transparency (which is already required of the board) is just as or more important than the decision to pay or not pay the board.
2) Transparently compensating any contributions that meet a well-defined bar is probably better than not compensating a contribution simply because it comes from the governing board.
3) If the transparency does not serve to control malicious behavior by the board (which is elected by the voting membership and funded by generous sponsors), then the corruption is already far enough along that the community has little to no future and sponsors would likely stop funding.

Reply:
Fortunately all transactions are transparent already on opencollective. What we might need in addition to that are transparent decisions. I think the Governing Board should explain in written form how it came to a financial decision and publish it somewhere.

Reply:
We might want to add a section about how we want to handle conflicts of interest of Board members. Maybe something along the lines of https://www.councilofnonprofits.org/tools-resources/conflicts-of-interest

Reply:
I think we should allow payments from the MapLibre Organization to Governing Board members for work which is not related to their services on the Board.

Reply:
@nyurik what do you think?

Reply:
Because even if we forbid payments to Board members from the MapLibre Organization completely, we still don't address the situation where Governing Board member A contracts with Company B to work on MapLibre Repository C. They will have a direct financial interest that the Board puts resources into Repository C...

Reply:
So my point is just let's do good conflict of interest management where we are transparent about what is going on and then we can also pay Board members to maintain a repository.

Reply:
Sigh, seems like there is no good solution.  If a board member works for a company that uses a project, that person will always have a conflict of interest in regards to that project, and will always steer funding towards that project at the expense of other projects.  I don't know of a good solution to have an impartial board. At the same time, I wouldn't want the board to become as ... irrelevant, as a Wikipedia board, so some vested interest is helpful I guess.

Reply:
I also want to have people on the Board which care about what the MapLibre Organization does.

I think it is fine to have people with a conflict of interest. It is just important to have a diversity of conflicts such that the Board members don't push the MapLibre Organization in a direction which would just benefit one actor.

Reply:
I think it could remain internal to a given board, but I think it would be valuable for each board member to have an informal understanding of others’ potential conflicts of interest to facilitate discussions when decisions to recusal are more nuanced. 

I’m not suggesting a change, just that a given board should be able to sort out its collective conflicts of interest and recuse appropriately. I don’t think the reasons need to (or should, for potentially sensitive reasons) be public, just the fact of the recusal.

Reply:
Let's continue this discussion in #100.